org.uqbar.project.wollok.lib/src/wollok/lang.wlk

```diff
@@ -44,6 +44,9 @@ class Exception {
 	
 	/** @private */
 	method createStackTraceElement(contextDescription, location) = new StackTraceElement(contextDescription, location)
+
+	/** Provides programmatic access to the stack trace information printed by printStackTrace() with full path files for linking */
+	method getFullStackTrace() native
```

Comment:
Do we need 2 methods now ? (getFullStackTrace() and getStackTrace() )

Couldn't this method replace the getStackTrace() ?